bitinformation_pipeline/bitinformation_pipeline.py

```diff
@@ -35,6 +35,39 @@ def get_user_input():
     return args
 
 
+def get_bit_coords(dtype_size):
```

Comment:
we ignore halfprecision, right?

Reply:
Well, it is just a draft. I'd like to add support for half precision as well. Any other datatypes we should think of right now?

Reply:
Actually I would like `get_bit_coords` to have `dtype` as an input, but that would mean we need to carry this information around when creating the `info_per_bits`-dict. Any opinions?

Reply:
What do you need dtype for?

Reply:
In the dataset you could put dtype as an attribute

Reply:
> Actually I would like get_bit_coords to have dtype as an input, but that would mean we need to carry this information around when creating the info_per_bits-dict. Any opinions?

Does `len(information_content)` not work for you? I remember we said to ignore `int` `types` for now and then I dont see why we need `dtype`.

Reply:
Yes, it is fine for now. I just thought a little bit more about the future ;)